files/image_config/sysctl/sysctl-net.conf

```diff
@@ -15,6 +15,7 @@ net.ipv4.conf.all.arp_announce=1
 net.ipv4.conf.all.arp_filter=0
 net.ipv4.conf.all.arp_notify=1
 net.ipv4.conf.all.arp_ignore=2
+net.ipv4.ip_local_reserved_ports=50051,50052
```

Comment:
Here is current SONiC auto allocate port range: 32768    60999
If any other SONiC service in this range, also need add to here.

admin@vlab-01:~$ sudo sysctl -a | grep ip_local_
**net.ipv4.ip_local_port_range = 32768    60999**
net.ipv4.ip_local_reserved_ports =
admin@vlab-01:~$